docs/LINUX.md

```diff
@@ -4,10 +4,9 @@ The LogDNA Agent for Linux collects log data from your Linux environment. The ag
 
 ## Table of Contents
   * [Considerations](#considerations)
-  * [Installation: fresh installs](#Installation)
```

Comment:
The anchor links are incorrect, it should be:
- Instead of `#Installation` should be `#installation-first-time-installations` 
- Instead of `#upgrademigration`, it should be `#upgrading-migrating-from-legacy-linux-agent`

Reply:
Thank you, @jorgebay !